homework/nwd-nnw/nwdNww.hpp

```diff
@@ -1,11 +1,35 @@
-#pragma once
-
-int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
-}
-
-int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
-}
+#pragma once
+
+int NWD(int lhs, int rhs) {
+    // TODO: Implement me :)
+    int pom;
+    
+if(lhs == 0 && rhs == 0){
+    return 0;
+}    
+else{
+    lhs=abs(lhs);
+    rhs=abs(rhs);
+        
+        while(rhs!=0)

```

Comment:
Add space between operations like '+' '-' '!=' '==' etc...
Add space before and after while, if, else etc...